modules/nf-core/anota2seq/anota2seqrun/meta.yml

```diff
@@ -231,11 +264,15 @@ output:
           type: file
           description: dump of R SessionInfo
           pattern: "*.log"
+          ontologies: 
+            - edam: "http://edamontology.org/format_2330" # Log
```

Comment:
this is the entry for text files, so either we change it or the comment should be `# Textual format`